### ISSUE TITLE
Errors when bash run.sh

### ISSUE THREAD

**Issue by Zecheng-Tang** [google-research-datasets/clang8#2]
Hi, thanks for your great work. 
Then, I  run the following command, it works.
```
echo "Running a test..."
python -m prepare_clang8_dataset_test
```
However, when I run the following command, there are some error reports.
```
python -m prepare_clang8_dataset \
  --lang8_dir="${LANG8_DIR}" \
  --tokenize_text='True' \
  --languages='ru,de,en'
```

![image](https://user-images.githubusercontent.com/59752775/129838939-e321cd05-81ea-4f60-99a9-d9cad6fb4f8d.png)
Anyway, the

**Comment by ekQ** [comment]
I would first check that the target files have been downloaded successfully by making sure that they contain the correct number of lines, see: https://github.com/google-research-datasets/clang8#data-format

**Comment by DincyDavis** [comment]
> I would first check that the target files have been downloaded successfully by making sure that they contain the correct number of lines, see: https://github.com/google-research-datasets/clang8#data-format

Yeah, I got the same error and the reason was target files were not downloaded properly, it was just 1 kb files instead of original size, make sure Git Large File Storage installed as mentioned in the steps and try once.

**Comment by mrqorib** [comment]
Run `git lfs pull` before the `./run.sh`
